### ISSUE TITLE
The problem that training requires 2000 epochs to converge 

### ISSUE THREAD

**Issue by xhhlll** [dbolya/yolact#722]
In my own training set, it only takes 50 epochs to train yolact on mmdetection to achieve mAP=85, but it takes 2000 epochs to get the same accuracy in the author's code. I count the iterations as epochs

**Comment by xqh-code** [comment]
> In my own training set, it only takes 50 epochs to train yolact on mmdetection to achieve mAP=85, but it takes 2000 epochs to get the same accuracy in the author's code. I count the iterations as epochs
How do you see how many rounds, I see only iterations in the code
